util/encode_modifiers.exs

```diff
@@ -11,11 +12,18 @@ defmodule ModifierEncoder do
     platform_modifiers: @native.platform_modifiers
 
   defmacro quote_argv() do
-    Code.string_to_quoted!(hd(System.argv()))
+    Code.string_to_quoted!(
+      hd(System.argv())
+      |> String.replace(~r'///', "")
+    )
   end
 
   def encode() do
-    IO.write(Phoenix.HTML.Safe.to_iodata(quote_argv().modifiers) |> IO.iodata_to_binary)
+    IO.write(
+      Phoenix.HTML.Safe.to_iodata(quote_argv().modifiers)
+      |> IO.iodata_to_binary
+      |> HtmlEntities.decode
```

Comment:
Same here. I also prefer to include the `()` for function calls with 0-arity as its more explicit; we should probably enforce this via `mix format` and Credo checks in the future.

```suggestion
      quote_argv().modifiers
      |> Phoenix.HTML.Safe.to_iodata()
      |> IO.iodata_to_binary()
      |> HtmlEntities.decode()
```